iib/workers/tasks/build.py

```diff
@@ -1426,8 +1475,31 @@ def _add_ocp_label_to_index(ocp_version, temp_dir, dockerfile_name):
     :param str dockerfile_name: the dockerfile name.
     """
     with open(os.path.join(temp_dir, dockerfile_name), 'a') as dockerfile:
-        label = f'LABEL com.redhat.index.delivery.version="{ocp_version}"'
+        label = f'LABEL {label_key}="{label_value}"'
         dockerfile.write(f'\n{label}\n')
         log.debug(
             'Added the following line to %s: %s', dockerfile_name, label,
         )
+
+
+def _validate_distribution_scope(resolved_distribution_scope, distribution_scope):
+    """
+    Validate distribution scope is allowed to be updated.
+
+    :param str resolved_distribution_scope: the distribution_scope that the index is for.
+    :param str distribution_scope: the distribution_scope that has been requested the index is for.
+    :return: the valid distribution_scope
```

Comment:
We can use plain English here instead of variable names :smile: 
```suggestion
    :return: the valid distribution scope
```